src/__tests__/clowder.test.ts

```diff
@@ -212,27 +242,30 @@ describe('Clowder Class', () => {
     expect(nextItem?.id).toMatch(/^(0|1)$/);
   });
 
-  it('should select an unvalidated item if no validated items remain', () => {
+  it('should return an item from missing if no validated items remain and returnUndefinedOnExhaustion is false', () => {
     const clowderInput: ClowderInput = {
       cats: {
         cat1: { method: 'MLE', theta: 0.5 },
+        cat2: { method: 'EAP', theta: -1.0 },
       },
       corpus: [
-        createMultiZetaStimulus('0', [createZetaCatMap([])]),
-        createMultiZetaStimulus('1', [createZetaCatMap(['cat1'])]),
-        createMultiZetaStimulus('2', [createZetaCatMap([])]),
+        createMultiZetaStimulus('0', [createZetaCatMap(['cat2'])]), // Validated for cat2
+        createMultiZetaStimulus('1', [createZetaCatMap(['cat2'])]), // Validated for cat2
+        createMultiZetaStimulus('2', [createZetaCatMap(['cat2'])]), // Validated for cat2
```

Comment:
Can we make one of these items unvalidated? I want to make sure that the unvalidated items wind up in `missing` also.

```suggestion
        createMultiZetaStimulus('1', [createZetaCatMap(['cat2'])]), // Validated for cat1
        createMultiZetaStimulus('2', [createZetaCatMap([])]), // Unvalidated
```